grid-cli/cmd/deploy_kubernetes.go

```diff
@@ -186,7 +186,7 @@ var deployKubernetesCmd = &cobra.Command{
 			masterNode = uint32(nodes[0].NodeID)
 		}
 		master.Node = masterNode
-		if workersNode == 0 && len(workers) > 0 {
+		if len(workersNode) < len(workers) && len(workers) > 0 {
 
 			filter, disks, rootfss := filters.BuildK8sFilter(
```

Comment:
buildK8sFilter should no longer expect the number of workers, it should filter the resources needed for only one node

Reply:
this comment is not addressed yet